File: cpp/evenprime.cpp
#include <iostream>
using namespace std;

int main()
{
    int a;
    cout<<"enter a number";
    cin>>a;
    cout<<(a%2==0? "even" : "odd") <<endl;
    for(int i=2; i<=a/2; ++i){
        if(a%i == 0)
        {   cout<<"not prime"; 
            break;
        }
        else
        {   cout<<"prime"; 
            break; 
        }
    }
}

File: old python files/evenprime.py
x = int(input("Enter a number- "))
if x%2 == 0:
    print(format(x)+" is even")
else:
    print(format(x)+" is odd")
n = int(x/2)
if x > 1:
    for i in range(2,n):
        if n%i == 0:
            print(format(x)+" is not prime")
            break
        else:
            print(format(x)+" is prime")
else:
    print(format(x)+"is not prime")

